packages/front-end/pages/metrics.tsx

```diff
@@ -42,10 +44,12 @@ const MetricsPage = (): React.ReactElement => {
   const { getUserDisplay } = useUser();
 
   const permissions = usePermissions();
+  const { apiCall } = useAuth();
 
   const tagsFilter = useTagsFilter("metrics");
 
   const [showArchived, setShowArchived] = useState(false);
+  const [recentlyArchived, setRecentlyArchived] = useState<string[]>([]);
```

Comment:
Should we use a Set instead of an array?  The current filter behavior is O(n^2) so if someone has a lot of metrics, that could get expensive.

Reply:
Sure, updated. I didn't initially because the setter hooks look a lot funkier to read for not much performance benefit.